src/main.c

```diff
@@ -31,8 +33,12 @@
 int
 main(int argc, char** argv)
 {
-  hiwire_setup();
```

Comment:
I blame this for your test failures

Reply:
Yes that looks like a problem. I wonder why it worked locally.